examples/wistia/src/css/cf-widget-api.css

```diff
@@ -0,0 +1,473 @@
+/* Contentful Widgets */
```

Comment:
This looks like a hard copy of `cf-widget-api.css`.

It would be more intuitive if you would just reference it or let a task to build the widget copy it. Check out some of the other examples for inspiration :)